From: Alex Gonzalez <alexg@balena.io>
Date: Mon, 30 Sep 2024 17:05:11 +0200
Subject: [PATCH] iot-gate-imx8: add placeholder for DTB IVT offset to
 environment

Upstream-status: Inappropriate [configuration]

Rebased for u-boot-compulab 2023.04, where the environment moved from
include/configs/iot-gate-imx8.h to include/configs/cpl-imx8m-mini.h.

Signed-off-by: Alex Gonzalez <alexg@balena.io>
---
 include/configs/cpl-imx8m-mini.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/cpl-imx8m-mini.h b/include/configs/cpl-imx8m-mini.h
index 5ffcffe120d8..067630c1474e 100644
--- a/include/configs/cpl-imx8m-mini.h
+++ b/include/configs/cpl-imx8m-mini.h
@@ -114,6 +114,7 @@
 	"mmcautodetect=yes\0" \
 	"root_opt=rootwait rw\0" \
 	"ivt_offset="__stringify(KERNEL_SIGN_IVT_OFFSET)"\0" \
+	"dtb_ivt_offset="__stringify(DTB_SIGN_IVT_OFFSET)"\0" \
 	"bootargs_common="__stringify(ENV_CONSOLE)" net.ifnames=0 "__stringify(BALENA_STAGE2)" \0" \
 	"emmc_ul=setenv iface mmc; setenv dev 2; setenv part 1;" \
 	"setenv bootargs ${bootargs_common} ${resin_kernel_root} ${root_opt} ${os_cmdline};\0" \
